src/plugins/data/public/search/session/sessions_client.ts

```diff
@@ -0,0 +1,91 @@
+/*
+ * Licensed to Elasticsearch B.V. under one or more contributor
+ * license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright
+ * ownership. Elasticsearch B.V. licenses this file to you under
+ * the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import { PublicContract } from '@kbn/utility-types';
+import { HttpSetup } from 'kibana/public';
+import type { SavedObject, SavedObjectsFindResponse } from 'kibana/server';
+import { BackgroundSessionSavedObjectAttributes, SearchSessionFindOptions } from '../../../common';
+
+export type ISessionsClient = PublicContract<SessionsClient>;
+export interface SessionsClientDeps {
+  http: HttpSetup;
+}
+
+/**
+ * CRUD backgroundSession SO
+ */
+export class SessionsClient {
```

Comment:
This is existing code moved from `src/plugins/data/public/search/session_service.ts` 

Session service was split into two services: 
1. `SessionsClient` (just SO CRUD)
2. `SessionSevice` (all that is related to state management of current session)